### PR TITLE
Multiplied mobile_robot size by 2

### PR DESCRIPTION
**Pull request by EnzoSigma**

### CODE REVIEW

File: fc_files/mobile_robot.fcmacro [hunk 1]
```diff
@@ -91,28 +91,28 @@ def make_mobile_robot():
...
-                             Placement(Vector(150.0, -100.0, -5.0), rotx(180.0)))
+                             Placement(Vector(300.0, -200.0, -5.0), rotx(180.0)))
 
     joint_hub_bl = add_joint(robot, 'undercarriage-hub_bl',
-                             Placement(Vector(150.0, 100.0, -5.0), roty(180.0)))
+                             Placement(Vector(300.0, 200.0, -5.0), roty(180.0)))
 
     joint_wheel_hr = add_joint(robot, 'hub_hr-wheel_hr',
-                               Placement(Vector(5.0, 0.0, 45.0), rotx(-90.0)))
+                               Placement(Vector(0.0, 20.0, 90.0), rotx(-90.0)))
```
Comment (galou) [inline fc_files/mobile_robot.fcmacro:106]: These values are not doubled. Are you sure?

**Review by EnzoSigma — COMMENTED** [review]
Add files for a mobile robot, a mobile manipulator and a manipulator with end-effector

**Comment by galou** [comment]
The second commit should have have been integrated into another PR.

**Comment by galou** [comment]
What is `manipulator_end_effector`? What does it contain. We already have `robot_5r-complete`.

**Pull request closed without merging**